recipes/speex/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export PKG_CONFIG_PATH=${PREFIX}/lib/pkgconfig
```

Comment:
This should be already set: https://conda.io/docs/building/environment-vars.html#environment-variables-set-during-the-build-process
could you please remove it?